### ISSUE TITLE
/click DominosActionButton<X> not working in macros

### ISSUE THREAD

**Issue by silasjmatson** [tullamods/Dominos#893]
I tested this with only Dominos & DominosConfig loaded.

I'm not sure if this is a skill issue on my part, but I have 2 macros, the goal of which is to be able to only update a single equip macro when getting gear upgrades, and not having to update all my other shield macros. This method has worked in the past with Bartender4, I got the idea from [this youtube video](https://youtu.be/E1_Nkl1V2gc?t=480).

1hander+shield equip macro
```
/equip Battlesmasher of the Tiger
/equip Furen's Favor
```

I then placed that macro in an action bar, specifically `DominosActionButton120`. (I found the button name via `/fstack`) I can click the button with my mouse and by typing `/click DominosActionButton120` in chat, and it equips the axe and shield as expected.

However when attempting to add this to my shield wall macro, my axe and shield are not equipped. Defensive stance goes off, so the macro is running.

Shield Wall macro
```
#showtooltip Shield Wall
/startattack [combat,harm,nodead]
/stopcasting [noworn:Shields]
/click DominosActionButton120
/cast Defensive Stance
/cast Shield Wall
```

If I replace `/click DominosActionButton120` with the contents of my 1hand equip macro, it works as expected, so this isn't a huge issue, but the convenience of only needing to update 1 macro on a weapon or shield upgrade was kinda nice.

**Addon Version**
11.0.4-classic

**World of Warcraft Version**
Classic Season of Discovery

**Comment by Tuller** [comment]
What happens if you add an explicit button, like the following?
`/click DominosActionButton120 LeftButton`

**Comment by silasjmatson** [comment]
I get the same results using `/click DominosActionButton120 LeftButton`

I just tested the same thing with no Addons loaded, and the macro doesn't work using the default blizzard buttons (e.g. `ActionButton10`, `MultiBarLeftButton12`), so I think it may be likely that Blizzard changed something with macros in the year or so since I had this working in Bartender4, and it just may not be possible anymore.

**Comment by silasjmatson** [comment]
After some more research, this is definitely an intended Blizzard change to prevent circumventing the macro size limit by chaining with `/click`: https://us.forums.blizzard.com/en/wow/t/please-fix-click-functionality/1967168/5
